src/main/java/org/jboss/set/overview/ejb/Aider.java

```diff
@@ -385,4 +388,27 @@ public void updatePayloadData() {
         }
         return null;
     }
+
+    public static List<VersionUpgrade> getComponentUpgrades(String component, String tag1, String tag2) {
+        try {
+            StreamComponent streamComponent = aphrodite.getStream(EAP70ZSTREAM).getComponent(component);
```

Comment:
Why only EAP70ZSTREAM ? Should it also be parameterized ?

Reply:
well at the moment I just need "an EAP stream" and the components are more or less the same across streams

Reply:
This is component definition for 7.0.z https://github.com/jboss-set/jboss-streams/blob/master/streams.json#L10974-L10984 leads to http://localhost:8080/prbz-overview/rest/api/tags/jbossas-jboss-eap7

This is component definition for 7.3.z https://github.com/jboss-set/jboss-streams/blob/master/streams.json#L3903-L3913, This expects something like http://localhost:8080/prbz-overview/rest/api/tags/wildfly-wildfly

When there is a new 7.4.z stream, we will probably stick to the component name convention to 7.3.z stream. So, could  you update this to 7.3.z stream ?